src/generator/www/js/main.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var formData = new FormData(form);
+ 
+  formData.append( 'sessionfile', $( '#sessionfile' )[0].files[0] );
+  formData.append( 'speakerfile', $( '#speakerfile' )[0].files[0] );
+  formData.append( 'trackfile', $( '#trackfile' )[0].files[0] );
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [There should be no spaces inside this paren.](https://www.codacy.com/app/triplez/open-event-webapp/file/2710866493/issues/source?bid=3233857&fileBranchId=3344140#l14)